tests/tacacs/test_ro_user.py

```diff
@@ -105,6 +111,8 @@ def test_ro_user_banned_command(localhost, duthosts, rand_one_dut_hostname, cred
     # Run as readonly use the commands allowed by sudoers file
     commands = [
             'sudo shutdown',
+            # all commands under the config tree
+            'sudo config'
```

Comment:
Do you prohibit below command? if yes, please add it as negative case.
```
sudo ipintutil <option>
```

Reply:
We don't prohibit `sudo ipintutil <option>`
The command can have following options
 `sudo ipintutil -a ipv6`
 ` sudo ipintutil -a ipv4` -> default
 `sudo ipintutil -n asic0 `

Reply:
Then add some positive cases in `commands_direct`?

Reply:
added in latest commit